src/react-elastic-carousel/components/Carousel.js

```diff
@@ -231,44 +234,54 @@ class Carousel extends React.Component {
     const { width: sliderContainerWidth } = sliderContainerNode.contentRect;
     // update slider container width
     // disable animation on resize see https://github.com/sag1v/react-elastic-carousel/issues/94
-    this.setState({ sliderContainerWidth, transitionMs: 0 }, () => {
-      // we must get these props inside setState (get future props because its async)
-      const {
-        onResize,
-        verticalMode,
-        itemsToShow,
-        children
-      } = this.getDerivedPropsFromBreakPoint();
+    const { sidePadding } = this.getDerivedPropsFromBreakPoint();
+    const containerWidth = sliderContainerWidth - sidePadding * 2;
```

Comment:
Should we support `sidePadding` for `verticalMode` as well?
I think this one is trickier as we have some logic to set the `height` and we don't rely on default styling like we do with the `width`.

If we do decide to support `verticalMode`, i think we need a new name as "side" will lose the context.
BTW, why we call it side"Padding" when we actually use margins? maybe we shouldn't use the words padding nor margin (implementation details?) and just use words like space or gap or something similar?

Reply:
I don't know how to deal with the vertical mode yet... maybe we can do it later, but for now I'll add "incompatible with verticalMode" on the prop's description?
I can name it `outerSpacing` ? You're right, "padding" doesn't make a lot of sense now that I'm using margins...

Reply:
(done)